OrbitGl/TracepointTrack.cpp

```diff
@@ -53,13 +69,31 @@ void TracepointTrack::UpdatePrimitives(uint64_t min_tick, uint64_t max_tick, Pic
 
   const Color kWhite(255, 255, 255, 255);
 
-  for (auto it = tracepoints.lower_bound(min_tick); it != tracepoints.upper_bound(max_tick); ++it) {
-    uint64_t time = it->first;
-    if (time < max_tick) {
+  const Color kGreenSelection(0, 255, 0, 255);
+
+  if (!picking) {
+    for (auto it = tracepoints.lower_bound(min_tick); it != tracepoints.upper_bound(max_tick);
+         ++it) {
+      uint64_t time = it->first;
       Vec2 pos(time_graph_->GetWorldFromTick(time), pos_[1]);
       batcher->AddVerticalLine(pos, -track_height, z, kWhite);
-    } else {
-      return;
+    }
+  }
+
+  else {
+    constexpr const float kPickingBoxWidth = 9.0f;
```

Comment:
constexpr float kPickingBoxWidth = 9.0f;
constexpr float kPickingBoxOffset = kPickingBoxWidth / 2.0f;

Reply:
Thank you for noticing that!